backend/common/position.go

```diff
@@ -197,3 +197,33 @@ func ConvertPGParserErrorCursorPosToPosition(cursorPos int, text string) *storep
 		Column: int32(column),
 	}
 }
+
+// ConvertLineToGitHubAnnotationPosition converts position to GitHub annotation position which
+// line starts by 1.
+func ConvertLineToGitHubAnnotationPosition(line int) int {
```

Comment:
If they're in the same logic, maybe we can only use one unified func like `ConvertAcionLine`?

Reply:
Maybe, we can derive them later while we considering column